src/bundle/Controller/SectionController.php

```diff
@@ -490,9 +507,25 @@ private function getSectionsNumbers(array $sections): array
      * @param \eZ\Publish\API\Repository\Values\Content\Section $section
      *
      * @return bool
+     *
+     * @throws \eZ\Publish\API\Repository\Exceptions\InvalidArgumentException
      */
     private function canUserAssignSectionToAnyContent(Section $section): bool
```

Comment:
naming, but something like `canUserAssignSectionToSomeContent` given a true here is more like a "maybe", while a false is pretty accurate.